src/react/components/pages/editorPage/editorPage.tsx

```diff
@@ -540,6 +558,41 @@ export default class EditorPage extends React.Component<IEditorPageProps, IEdito
             case ToolbarItemName.RemoveAllRegions:
                 this.canvas.current.confirmRemoveAllRegions();
                 break;
+            case ToolbarItemName.ActiveLearning:
+                await this.predictRegions();
+                break;
+        }
+    }
+
+    private predictRegions = async (canvas?: HTMLCanvasElement) => {
+        canvas = canvas || document.querySelector("canvas");
+        if (!canvas) {
+            return;
+        }
+
+        // Load the configured ML model
+        if (!this.activeLearningService.isModelLoaded()) {
+            let toastId: number = null;
+            try {
+                toastId = toast.info(strings.activeLearning.messages.loadingModel, { autoClose: false });
+                await this.activeLearningService.ensureModelLoaded();
+            } catch (e) {
+                toast.error(strings.activeLearning.messages.errorLoadModel);
```

Comment:
Shouldn't we call first toast.dismiss(toastId) before the toast.error ?

Reply:
Either way they are called at the same time and since there is animation you see both for a split second anyway.  Keep here allows us to keep it within the finally block.